activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+*   Do not set sql_mode if specify `strict: :default` explicitly.
+
+        # database.yml
+        production:
+          adapter: mysql2
+          database: foo_prod
+          user: foo
+          strict: :default
+
+    *Ryuta Kamizono*
+
```

Comment:
Thanks for the changelog entry! The grammar is a little off and is confusing to read. Can you update the language to say:

"Do not set `sql_mode` if `strict: :default` is specified"

And then put the database.yml example in back ticks so it's a code block. Like

```
# database.yml
  production:
  adapter: mysql2
  database: foo_prod
  user: foo
  strict: :default
```